From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 28 Oct 2021 01:27:53 +0200
Subject: [PATCH] Disable HTML docs, install only man pages

HTML docs are big and they are available only anyway.

Patch-Source: https://git.alpinelinux.org/aports/tree/main/postgresql16/disable-html-docs.patch

--- a/doc/src/sgml/Makefile
+++ b/doc/src/sgml/Makefile
@@ -26,9 +26,9 @@
 include $(top_builddir)/src/Makefile.global
 
 
-all: html man
+all: man
 
-distprep: html distprep-man
+distprep: distprep-man
 
 
 ifndef DBTOEPUB
@@ -220,14 +220,14 @@
 ## Install
 ##
 
-install: install-html install-man
+install: install-man
 
 installdirs:
-	$(MKDIR_P) '$(DESTDIR)$(htmldir)'/html $(addprefix '$(DESTDIR)$(mandir)'/man, 1 3 $(sqlmansectnum))
+	$(MKDIR_P) $(addprefix '$(DESTDIR)$(mandir)'/man, 1 3 $(sqlmansectnum))
 
 # If the install used a man directory shared with other applications, this will remove all files.
 uninstall:
-	rm -f '$(DESTDIR)$(htmldir)/html/'* $(addprefix  '$(DESTDIR)$(mandir)'/man, 1/* 3/* $(sqlmansectnum)/*)
+	rm -f $(addprefix  '$(DESTDIR)$(mandir)'/man, 1/* 3/* $(sqlmansectnum)/*)
 
 
 ## Install html
